graphql-dgs-client/src/test/java/com/netflix/graphql/client/GraphQLResponseJavaTest.java

```diff
@@ -34,7 +34,7 @@
     RestTemplate restTemplate = new RestTemplate();
     MockRestServiceServer server = MockRestServiceServer.bindTo(restTemplate).build();
     String url = "http://localhost:8080/graphql";
-    DefaultGraphQLClient client = new DefaultGraphQLClient(url);
+    DefaultGraphQLClient client = new DefaultGraphQLClient(url, null);
```

Comment:
this shouldn't be necessary. Don't change existing Java code, because that indicates it will be breaking for users.